sourcegraph/sourcegraph
# Update HTML and CSS for new deploy docs

Closes https://github.com/sourcegraph/sourcegraph/pull/41223 -  Make HTML and CSS on the index page production-quality (Beatrix)

This PR:
- Updates HTML and CSS on the index page production-quality
- Updates HTML and CSS on the AWS One-Click page
- Aligns with other doc components / styles
- Works with both dark and light themes
- Make contents accessible

Light theme:
![image](https://user-images.githubusercontent.com/68532117/188252783-23c40b09-a9d2-45b3-984c-e14a34f43b53.png)

![image](https://user-images.githubusercontent.com/68532117/188253157-636c91d5-be9c-4e02-a5e2-d4bd622b5fc5.png)


Dark theme:
![image](https://user-images.githubusercontent.com/68532117/188252789-a218db6f-8f4e-4837-9472-188731c7fa0d.png)

![image](https://user-images.githubusercontent.com/68532117/188253170-1859dff8-4202-49d1-bb45-8cd6c5caaa18.png)

Next:
- [x] Run through and update deployment docs for each deployment type
- [ ] Add warning note to Kubernetes deployment docs
- [ ] Update link to Sourcegraph Cloud deployment doc when it's ready
- [x] Create Azure deployment docs for Docker Compose? 


## Test plan



To view in local dev,

1. Check out this branch
2. Run yarn docsite:serve from the repo root
3. Point your browser at http://localhost:5080/next
